### ISSUE TITLE
regex property schema error

### ISSUE THREAD

**Issue by gerrycampion** [cdisc-org/cdisc-rules-engine#524]
regex should be added as a top-level general property in Operator.json.
Without it you get a schema warning on a rule like this:

```yaml
    - name: RDOMAIN
      operator: does_not_equal_string_part
      regex: ".{4}(..).*"
      value: dataset_name
```

"Property regex is not allowed."

Refer to CG0334 for full rule.
